antd/src/main/scala/demo/App.scala

```diff
@@ -77,8 +79,13 @@ object App {
               .setDataIndex("name")
               .setKey("name")
               .setRender((_, tableItem, _) => Tag(tableItem.name).rawElement),
-            ColumnType[TableItem].setTitle("Age").setDataIndex("age").setKey("age"),
-            ColumnType[TableItem].setTitle("Address").setDataIndex("address").setKey("address")
+            // TODO: is a varargs constructor missing here?
+            ColumnGroupType[TableItem](
+              scala.scalajs.js.Array(
```

Comment:
varargs?

Reply:
actually, no. The reason is that all *optional* things get builder methods (which can be overloaded and/or renamed). All *required* props need to be supplied directly in the `apply` method. 

It's something that could be improved naturally, but it would need a step-wise typed builder thing otherwise